CHANGELOG/CHANGELOG-0.1.md

```diff
@@ -0,0 +1,31 @@
+# Changelog 0.1
+
+## 0.1.0 - 2022-06-16
+
+This is the first Theia release and it includes all Antrea flow visibility capabilities released as of 1.6.
+
+
+### Added
+
+- Helm support for Theia installation ([#15](https://github.com/antrea-io/theia/pull/15) [@yanjunz97]) ([#31](https://github.com/antrea-io/theia/pull/31) [@yanjunz97]) ([#34](https://github.com/antrea-io/theia/pull/34)  [@yanjunz97])
+  * Provide Helm charts for Theia components (Spark operator, Clickhouse, and Grafana)
+  * Allow for deploying Clickhouse on a persistent volume
+- Grafana dashboard: add a panel for visualizing denied connections ([#12](https://github.com/antrea-io/theia/pull/12) [@heanlan])
+  * Introoduce chord diagram for visualizing connections denied by network policies
+  * Reorganized Network policy dashboard to visualize both allowed and denied traffic
+- Policy recommendation via Spark job ([#16](https://github.com/antrea-io/theia/pull/16) [@dreamtalen])
```

Comment:
`theia` cli tool might also be worth mentioning:
- Theia Command-line Tool ([#25](https://github.com/antrea-io/theia/pull/25) [@dreamtalen][@yanjunz97][@annakhm])
   * Add theia command-line tool to provide access to Theia network flow visibility capabilities.
   * Introduce 3 commands for the NetworkPolicy Recommendation feature.

Reply:
Do we also want to include the following items?  

Added:
- the documentation PR #40 #45 
- github workflow #11 
- basic e2e test for NetworkPolicy Recommendation #27 

Fixed:
- FlowAggregator e2e test ANP deployment issue #42

Reply:
I prefere not to include PR #11 as it is about basic project CI infra and has nothing that pertains theia itself.
It makes sense to add a bullet for documentation and one for e2e tests.